### ISSUE TITLE
Python 2 only?

### ISSUE THREAD

**Issue by frafra** [opencultureconsulting/openrefine-client#27]
`README.md` mentions Python 2, but not Python 3. Is this project meant to be used only with Python 2.x?

**Comment by felixlohmeier** [comment]
Yes, that is unfortunately the current status. There is an unfinished attempt to port it to Python 3, see https://github.com/wolfv/openrefine-client/tree/py3_port and discussion in pull request https://github.com/opencultureconsulting/openrefine-client/pull/8
